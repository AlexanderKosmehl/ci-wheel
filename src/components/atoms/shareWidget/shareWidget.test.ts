import { describe, expect, it } from 'vitest';
import generateShareWidget from './shareWidget';

describe('generateShareWidget', () => {
  const shareWidget = generateShareWidget();

  it('generates component correctly', () => {
    expect(shareWidget).toMatchSnapshot();
  });
});
