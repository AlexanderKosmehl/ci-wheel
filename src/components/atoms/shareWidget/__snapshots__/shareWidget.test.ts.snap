// Vitest Snapshot v1

exports[`generateShareWidget > generates component correctly 1`] = `
<div>
  <button
    class="_button_6c4ac5"
  >
    <img
      class="_shareIcon_6c4ac5"
      src="/src/assets/share-icon.svg"
    />
    <span>
      Teilen
    </span>
  </button>
</div>
`;
